tests/foreman/cli/test_import.py

```diff
@@ -19,33 +19,34 @@
 from robottelo.cli.user import User
 
 
-def csv_to_dataset(csv_file):
+def csv_to_dataset(csv_files):
     """Process a remote CSV file.
 
-    Read a remote CSV file, process it and return it.
+    Read the remote CSV files, and return a list of dictionaries for them
 
-    :param csv_file: A string. The path to a CSV file that resides
+    :param csv_files: A list of strings. The paths to CSV files that reside
         on a remote server.
 
-    :returns: A dictionary holding the contents of the CSV file.
+    :returns: A dictionary holding the contents of the CSV files.
```

Comment:
This docstring contains quite a bit of incorrect and weirdly worded information. How about this?

    """Process and return remote CSV files.

    :param csv_files: A list of strings, where each string is a path to a CSV
        file on the remote server.
    :returns: A list of dicts, where each dict holds the contents of one CSV
        file.

    """